baseapp/baseapp.go

```diff
@@ -412,11 +413,7 @@ func (app *BaseApp) CheckTx(txBytes []byte) (res abci.ResponseCheckTx) {
 		Log:       result.Log,
 		GasWanted: result.GasWanted,
 		GasUsed:   result.GasUsed,
-		Fee: cmn.KI64Pair{
```

Comment:
Why are we getting rid of this? Presumably Tendermint needs to know about the fee?

Reply:
Fees were removed. 
from the changelog:
`[abci] Removed Fee from ResponseDeliverTx and ResponseCheckTx`

Reply:
@ValarDragon thoughts on the above?

Reply:
No reason to add priority right now. We're not ready to support that at the mempool level, and we won't be for awhile. We discussed that priority is something we want to add as a way to test protobuf upgradability on a test net, not right now.

Reply:
Gotcha.

Reply:
OK, I think it's a little unusual to delay ABCI updates because Tendermint won't use the value yet, but nbd.